### PR TITLE
Added release-management.md

### PR DESCRIPTION
**Pull request by ChandanSagar**
Added release-management.md which contains the release tagging process steps.
Signed-off-by: Chandan Sagar Pradhan [chandan.pradhan@mayadata.io](chandan.pradhan@mayadata.io)

### CODE REVIEW

File: contribute/design/release-management.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- openebs/cstor
+- openebs/istgt
+- openebs/jiva
+- openebs/ndm
+- openebs/external-storage
+- openebs/velero-plugin
+- openebs/maya
+
+The e2e pipeline and exploratory tests are executed using the RC tagged images. Any issues identified during this RC testing are tagged as release blockers.
+After the issues are fixed and verified by the CI, next RC is triggered.(Example: 0.1.0-RC2)
```
Comment (kmova) [inline contribute/design/release-management.md]: n_ : to follow the release example above - make it 1.0.0-RC2

File: contribute/design/release-management.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+The above process is repeated till a RC has been arrived with all the release blockers fixed.
+After the final RC, a release tag is created. (Example 0.1.0) in the same order as the release candidate builds. The e2e pipeline and exploratory tests are repeated on the release tagged builds.
```
Comment (kmova) [inline contribute/design/release-management.md]: same as above.

File: contribute/design/release-management.md (hunk 1 above)
Comment (ChandanSagar) [reply contribute/design/release-management.md]: Done.

File: contribute/design/release-management.md (hunk 2 above)
Comment (ChandanSagar) [reply contribute/design/release-management.md]: Done.

**Pull request merged**